website/core/TabsConstants.js

```diff
@@ -19,6 +19,13 @@ const packageManagers = [
 ];
 const defaultPackageManager = 'npm';
 
+const androidLanguages = [
+  {label: 'Java', value: 'java'},
+  {label: 'Kotlin', value: 'kotlin'},
+];
+
+const defaultAndroidLanguage = 'kotlin';
```

Comment:
Are we sure the default should be Kotlin?

Reply:
fixed it